### ISSUE TITLE
Table validation results not presented properly

### ISSUE THREAD

**Issue by ghost** [tomschr/daps#660]
### Description

This only affects DAPS from `main` (as of faf5f0a):
When running into a table validation issue, DAPS will present the output of the table validation script, but remove all line breaks, rendering the output very hard to understand:

```
Fatal error: The following tables contain errors:
release-notes.xml:5504: <informaltable> Errors in table:   - Holes in row 2   - Holes in row 6   - Holes in row 8   - Holes in row 9   - Holes in row 10   - Holes in row 11    H oooooo     ------   B oooooo !!B ooooo    B oooooo   B oooooo   B oooooo !!B ooo      B oooooo !!B oooo   !!B ooo    !!B ooo    !!B ooooo    B oooooo   B oooooo   B oooooo   B oooooo 
--------------------------------
Document does not validate!
```

Normally, the output should look like this which is much easier to understand:

```
Fatal error: The following tables contain errors:
release-notes.xml:5504: <informaltable> Errors in table:
   - Holes in row 2
   - Holes in row 6
   - Holes in row 8
   - Holes in row 9
   - Holes in row 10
   - Holes in row 11

   H oooooo
     ------
   B oooooo
 !!B ooooo
   B oooooo
   B oooooo
   B oooooo
 !!B ooo
   B oooooo
 !!B oooo
 !!B ooo
 !!B ooo
 !!B ooooo
   B oooooo
   B oooooo
   B oooooo
   B oooooo 
--------------------------------
Document does not validate!
```

### Steps to reproduce problem

Validate (or try to build) a document with DAPS from `main`.
I used the SLES 12 SP2 RN (internal repo, sorry), SHA 6f3b66.

**Comment by ghost** [comment]
We do use `echo -e` there already...

https://github.com/openSUSE/daps/blob/62ebaebc2366f6e42f90a8740e8d91e013569c63/make/validate.mk#L76-L78

... possibly an issue that results from using a make variable to store the result earlier on?

**Comment by ghost** [comment]
fixed via [f1b18b0](https://github.com/openSUSE/daps/commit/f1b18b0f2be4da8d2d55245da12b95c598a4b5c3)
